### PR TITLE
Version Bump 0.7.11 Added Join Accept Decrypt + Decript packet with 32 bit counter

### PR DESCRIPTION
**Pull request by kalik1**
Thanks to @avbentem in issue #10 
some files should not be merged (like packet.json or .gitignore)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,19 +17,19 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/anthonykirby/lora-packet.git"
+    "url": "https://git.unidata.it/LORA/lora-packet"
```
Comment (anthonykirby) [inline package.json]: seriously?

Comment (kalik1) [reply package.json]: As i wrote it, it must not be merged. I needed it for validating project in my CI, but i forgot to remove.

**Comment by kalik1** [comment]
sorry for formatting changes, Webstorm IDE :)

File: lib/crypt.js [hunk 2]
```diff
@@ -3,6 +3,8 @@
 var util = require('./util.js')("lora-packet crypto");
 
 var CryptoJS = require("crypto-js");
+var aesCmac = require('node-aes-cmac').aesCmac;
```
Comment (anthonykirby) [inline lib/crypt.js]: not used?

File: lib/crypt.js [hunk 3]
```diff
@@ -3,6 +3,8 @@
 var util = require('./util.js')("lora-packet crypto");
 
 var CryptoJS = require("crypto-js");
```
Comment (anthonykirby) [inline lib/crypt.js]: not used?

File: lib/crypt.js (hunk 2 above)
Comment (kalik1) [reply lib/crypt.js]: confirmed note used. was for testing.

**Comment by anthonykirby** [comment]
hi Giorgio,
I think I've made the functional changes & published as 0.7.10.  I've left out the cosmetic changes & the ones that are probably just debug code.  Many thanks for the contribution - much appreciated!
Anthony

**Comment by kalik1** [comment]
Thanks for merging. Now let' work on Lorawan 1.1 :)

**Pull request closed without merging**